augur/cli/db.py

```diff
@@ -331,3 +332,188 @@ def check_pgpass_credentials(config):
         else:
             logger.info("Credentials found in $HOME/.pgpass")
 
+@cli.command('update-org-repositories')
```

Comment:
The reason to have this in the CLI is for sophisticated users to update their repositories when a large number of them have changed, without having to run all of augur. 

It does *not* duplicate existing functions.